.eslintrc

```diff
@@ -1,32 +1,33 @@
-{
-  "env": {
-    "browser": true,
-    "commonjs": true,
-    "es6": true,
-    "node": true,
-    "jest": true
-  },
-  "parser": "@babel/eslint-parser",
-  "plugins": ["react", "jest"],
-  "extends": [
-    "eslint:recommended",
-    "plugin:prettier/recommended",
-    "plugin:cypress/recommended",
-    "next/core-web-vitals"
-  ],
-  "parserOptions": {
-    "sourceType": "module",
-    "ecmaVersion": 2018
-  },
-  "settings": {
-    "react": {
-      "version": "detect"
-    }
-  },
-  "rules": {
-    "@next/next/no-html-link-for-pages": "off",
-    "linebreak-style": ["error", "unix"],
-    "require-atomic-updates": "off",
-    "react/jsx-uses-vars": 2
-  }
-}
+{
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es6": true,
+    "node": true,
+    "jest": true
+  },
+  "parser": "@babel/eslint-parser",
+  "plugins": ["react", "jest"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:prettier/recommended",
+    "plugin:cypress/recommended",
+    "next/core-web-vitals"
+  ],
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaVersion": 2018
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "rules": {
+    "@next/next/no-html-link-for-pages": "off",
+    "@next/next/no-document-import-in-page": "off",
+    "linebreak-style": 0,

```

Comment:
Why disable linebreak-style? Just curious, I've had "local" issues with this before but I wonder if disabling could end up causing issues with the Circle CI executor?

Reply:
I was to facilitate easier developing for everyone on Windows machines, but I have reverted it anyway